lib/experiments-page.php

```diff
@@ -138,6 +138,12 @@ function gutenberg_experiments_editor_settings( $settings ) {
 		'__experimentalEnableFullSiteEditing'   => $experiments_exist ? array_key_exists( 'gutenberg-full-site-editing', get_option( 'gutenberg-experiments' ) ) : false,
 
 	);
+
+	$gradient_presets = current( (array) get_theme_support( 'editor-gradient-presets' ) );
```

Comment:
We are doing some work around these APIs, e.g: automatically generate the styles, provide better hooks/API for blocks, etc...
I guess until this work is on a good path we should keep this API experimental e.g.:
```
$gradient_presets = current( (array) get_theme_support( '__experimental-editor-gradient-presets' ) );
```